.bazelrc

```diff
@@ -26,18 +26,16 @@ build --flag_alias=cross=//build/toolchains:cross_flag
 build --flag_alias=dev=//build/toolchains:dev_flag
 build --flag_alias=force_build_cdeps=//build/toolchains:force_build_cdeps_flag
 build --flag_alias=heavy=//build/toolchains:heavy_flag
-build --flag_alias=lintonbuild=//build/toolchains:nogo_flag
-build --flag_alias=nolintonbuild=//build/toolchains:nonogo_explicit_flag
 
 build:crdb_test_off --crdb_test_off
 build:cross --cross
 build:dev --dev
 build:force_build_cdeps --force_build_cdeps
 build:heavy --heavy
-build:lintonbuild --lintonbuild
-build:nolintonbuild --nolintonbuild
+build:lintonbuild --run_validations
```

Comment:
Just for my understanding, why is this changing?

Reply:
The API of `nogo` (changed)[https://github.com/bazel-contrib/rules_go/blob/master/go/nogo.rst] in the recent release we're updating to. [This commit](https://github.com/bazel-contrib/rules_go/commit/29d4e5dbcfa702c479311f0bb4a929a236666a46) updates `nogo` to use validation actions instead of baking `nogo` into the toolchain. It's actually a significant UX improvement.